### PR TITLE
make it easier for newbies to get started

### PR DESCRIPTION
**Pull request by sadams**
1. make it easier for newbies to get started
2. update deps and fix deprecation warnings for beforeHandler

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,8 +28,8 @@
   "license": "MIT",
   "devDependencies": {
     "fastify": "^2.0.0",
-    "fastify-jwt": "^0.9.0",
-    "fastify-leveldb": "^0.3.0",
+    "fastify-jwt": "^0.9.3",
+    "fastify-leveldb": "^1.0.0",
```
Comment (Eomm) [inline package.json]: I don't think this change is up to this PR, since it should only to clarify some pieces of code

**Comment by Eomm** [comment]
@sadams would you mind to complete this PR in order to land it later today? 💪

**Comment by sadams** [comment]
@Eomm i just tried to do this quickly but i'm on node@12 and leveldb wont build (https://github.com/Level/level/issues/148 - i though maybe i upgraded it because it didn't work otherwise, but not sure now). I am short on time today to change node version + test but will get to it soon. I will also add note in readme that it doesn't support node@12 (level needs to be v5+ to support to support node12+.

**Comment by Eomm** [comment]
For node 12 there is a PR ready https://github.com/fastify/fastify-auth/pull/47

I would like to release all this within the weekend 👍

**Comment by sadams** [comment]
@Eomm have pushed your suggested changes.

**Pull request merged**